src/atomate2/vasp/sets/base.py

```diff
@@ -398,20 +408,16 @@ def get_input_set(  # type: ignore
         structure, prev_incar, bandgap, ispin, vasprun, outcar = self._get_previous(
             structure, prev_dir
         )
-        incar_updates = self.get_incar_updates(
-            structure,
-            prev_incar=prev_incar,
-            bandgap=bandgap,
-            vasprun=vasprun,
-            outcar=outcar,
-        )
-        kpoints_updates = self.get_kpoints_updates(
-            structure,
-            prev_incar=prev_incar,
-            bandgap=bandgap,
-            vasprun=vasprun,
-            outcar=outcar,
-        )
+        prev_incar = prev_incar if self.inherit_incar else {}
```

Comment:
@utf To get the tests passing, we needed to disable INCAR inheritance. This PR (2177b0f1) adds a new keyword `inherit_incar: bool = None` to `VaspInputGenerator` which disables INCAR inheritance by default.

Reply:
Why do you need to disable inheritance for the tests to pass? Surely the test data can be changed?